controllers/articlesController.js

```diff
@@ -93,4 +82,154 @@ export default class PostController {
       });
     }
   }
+
+  /**
+   * @description It helps the user to fetch all of the created articles.
+   * @param  {object} req - The request object
+   * @param  {object} res - The response object
+   * @returns {object} It returns the request's response object
+   */
+
+  static async findAll(req, res, next) {
+    const { user: { id = null } = {} } = req;
+    const { author, favorited, tag } = req.query;
+    const { page = 1, limit = 20 } = req.query;
+    try {
+      const all = await Article.findAll({
+        offset: (Number(page) - 1) * Number(limit),
+        limit,
+        order: [["createdAt", "DESC"]],
+        where: { deletedAt: null },
+        attributes: { exclude: ["deletedAt"] },
+        include: [
+          {
+            model: User,
+            as: "author",
+            attributes: ["username", "bio", "image"],
```

Comment:
@Musigwa Don't you need the user's names?

Reply:
I think we do, I have consulted our UI pages and found that I should put it in. Thanks for that quick reminder, I'm implementing it in the near future commit.